operatoroverloading.dd

```diff
@@ -703,6 +703,98 @@ $(H2 $(LEGACY_LNAME2 ArrayOps, array-ops, Array Indexing and Slicing Operators O
 	These may be combined to implement multidimensional arrays.
 	)
 
+	$(P The code example below shows a simple implementation of a
+	2-dimensional array with overloaded indexing and slicing operators. The
+	explanations of the various constructs employed are given in the
+	sections following.)
+
+-------
+struct Array2D(E)
+{
+    E[] impl;
+    int stride;
+    int width, height;
+
+    this(int _width, int _height, E[] initialData = [])
```

Comment:
Underscores in front of the fields are somewhat normal, but not in front of arguments.
Maybe use `this.width` instead.